src/buildercore/config.py

```diff
@@ -116,11 +116,13 @@ def get_logger(name):
 
 #
 # settings
-# believe it or not but buildercore.config is NOT the place for user config
+# buildercore.config is NOT the place for user config
 #
 
 PROJECTS_FILES = ['projects/elife.yaml']
 
+PROJECT_FORMULAS = os.path.join(PROJECT_PATH, 'cloned-projects') # same path as used by Vagrant
```

Comment:
unclear name, looks like a list of formulas but it's actually a path to their folder